src/components/FullPageOfflineView.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import Icon from './Icon';
+import Text from './Text';
+import * as Expensicons from './Icon/Expensicons';
+import themeColors from '../styles/themes/default';
+import styles from '../styles/styles';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import ONYXKEYS from '../ONYXKEYS';
+import FullScreenLoadingIndicator from './FullscreenLoadingIndicator';
+import compose from '../libs/compose';
+
+const propTypes = {
+    /** Child elements */
+    children: PropTypes.node.isRequired,
+
+    isLoading: PropTypes.bool.isRequired,
```

Comment:
Actually, thinking about this some more... Why does this component need to manage a loading state? Isn't that a separate concern? Maybe we should keep it simple / concerned only with online/offline.

Reply:
Hmm, yea I don't know if/why it needs that. I think @marcaaron was the one to initially draft the pseudo-code for this component in the UX Patterns doc. It also had the error prop in there as well.

Reply:
Actually yea, we do need isLoading bc this component is a wrapper. We want to show the full page loading spinner with this component if this prop gets passed as true.

Reply:
Ok aaaactually after discussing 1:1, we're gonna take it out.

Reply:
Yeah, I'm not sure if those props really made sense - probably just oversight. If we need this view to have a loading spinner or display an error message we can add it later (though having trouble thinking of an example where it would). If you go offline and a view turned into this it would be pretty clear what happened 😄